admin/users/views.py

```diff
@@ -61,15 +61,11 @@ def delete(self, request, *args, **kwargs):
                     action_flag=flag
                 )
         except AttributeError:
```

Comment:
We usually try to limit the amount of code in a `try` block that isn't expected to raise an error.

Duck-typing is generally intended to determine whether or not an object is suitable based on the methods/properties/attributes it has, but any `User` object should have all of these.